guides/doc-Project_API/topics/con_oauth-authentication-overview.adoc

```diff
@@ -0,0 +1,15 @@
+[id="oauth-authentication-overview"]
+= OAuth authentication overview
+
+As an alternative to basic authentication, you can use limited OAuth 1.0 authentication.
+This is sometimes referred to as _1-legged OAuth_ in version 1.0a of the protocol.
```

Comment:
Needs technical ACK please.

Reply:
Good enough? > https://docs.oracle.com/en/cloud/paas/application-integration/adapter-builder/oauth-1.0-one-legged-authentication.html

Reply:
Technical ACK regarding what exactly? The version of the OAtuh spec?

Reply:
Yes, regarding the version and naming.

Reply:
I've got an ack from engineering that Foreman only supports OAuth 1.0a authentication.
What about the naming?

Reply:
I was just wondering about "1-legged", but an internet search reveals that it's a thing :D

Reply:
FYI, I had to rephrase it, because 1-legged is not used in the specification itself :D